waspc/src/Wasp/Generator/DbGenerator/Operations.hs

```diff
@@ -28,7 +32,7 @@ printJobMsgsUntilExitReceived chan = do
     J.JobOutput {} -> printJobMessage jobMsg >> printJobMsgsUntilExitReceived chan
     J.JobExit {} -> return ()
 
--- | Migrates in the generated project context and then copies the migrations dir back
+-- | Migrates in the generated project context, stores checksum, and then copies the migrations dir back
```

Comment:
checksum of what?

Reply:
you might even consider ommiting it since it is somewhat of an implementation detail.

Reply:
Prisma schema checksum... but yeah it may be too implementation specific. 🤔 May nix it